client/scss/components/_promo.scss

```diff
@@ -28,10 +68,102 @@
   }
 }
 
-.promo__intro {
+.promo__image {
+  display: block;
+  width: 100%;
+}
+
+.promo__defs {
```

Comment:
Relates to the svg filter, used on the --series promo

https://developer.mozilla.org/en-US/docs/Web/SVG/Element/defs